.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea
+.vscode
+.settings
+.classpath
+.project
```

Comment:
Could you add `*.class` to this -- we had some issues with this in another pull request